spec/ewb/extensions/IEC61968/Metering/PanDemandResponseFunction.yaml

```diff
@@ -0,0 +1,11 @@
+name: PanDemandResponseFunction
+description: PAN function that an end device supports, distinguished by 'kind'.
```

Comment:
todo: Missing ZBEX in descriptions. Search the repo for other uses of this. This applies to other extensions.

Reply:
Added to the lot, and also added the missing attribute to regulatingcontrol.